### ISSUE TITLE
Enhancements features doesn't appear on Garmin

### ISSUE THREAD

**Issue by Stelle-5** [Wassup789/garmin-connect-enhancements#2]
### Description

The plugin is installed and activated but on Garmin Connect we can't see the features. (Filter exercises, preview muscle group target, favorating exercices)

### Reproduction steps

https://github.com/user-attachments/assets/114c4953-c6b5-4def-8ea9-0e99f00dfc9f

### Which platform did you discover this bug?

Chrome Web Store

### Please provide the environment you discovered this bug

```true
Version of the plugin : v1.1.1
Browser : Brave Version 1.68.134 Chromium: 127.0.6533.88 (lastest)
OS : Windows 10 22H2
```

### Additional information

I've already tried to change the switch to another navigator (Firefox), and tried to install the plugin via Violentmonkey but the features doesn't appeared

**Comment by Wassup789** [comment]
Thanks for bringing this issue up.

There was a recent change by Garmin to the exercise selector (specifically for editing _**workouts**_) that caused the selector to break (editing **_activities_** still work as intended). I actually started work on this yesterday, plan to get a fix out by this week or early next week.

**Comment by Wassup789** [comment]
Fixed in [v1.1.2](https://github.com/Wassup789/garmin-connect-enhancements/releases/tag/v1.1.2), which just got published so look out for an update.
